pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1207,7 +1207,14 @@ private CompletableFuture<Void> delete(boolean failIfHasSubscriptions,
                         brokerService.deleteTopicAuthenticationWithRetry(topic, deleteTopicAuthenticationFuture, 5);
 
                         deleteTopicAuthenticationFuture.thenCompose(__ -> deleteSchema())
-                                .thenCompose(__ -> deleteTopicPolicies())
+                                .thenCompose(__ -> {
+                                    if (!this.getBrokerService().getPulsar().getBrokerService()
```

Comment:
Is only the system topic `__change_event` not required policies?

Reply:
` __transaction_buffer_snapshot` is also not requiring policies.

Reply:
I think about it carefully, and now it's reasonable(even if other system topics have policies). Because:

- Deleting `__change_event` deletes all the policies for the topic.
- Except for deleting the namespace, no other situation will delete `__change_event`.

Should we add some code-comment and a test for this?